src/components/input/input.spec.js

```diff
@@ -436,6 +436,59 @@ describe('md-input-container directive', function() {
     expect(el[0].querySelector("[ng-messages]").classList.contains('md-auto-hide')).toBe(false);
   }));
 
+  it('should set the animation class on the ngMessage properly', inject(function() {
+    var element = compile(
+      '<md-input-container>' +
+        '<input ng-model="inputVal">' +
+        '<div ng-messages>' +
+          '<ng-message id="requiredMessage" when="required">Field required</ng-message>' +
+        '</div>' +
+      '</md-input-container>'
+    );
+
+    var ngMessage = element.find('ng-message');
+    expect(ngMessage).toHaveClass('md-input-message-animation');
+  }));
+
+  it('should set the animation class on a transcluded ngMessage', function() {
```

Comment:
I'm curious if we could do a different/shorter test here.

Based on the issues this is supposed to fix, if we put it inside an autocomplete, or use the `ng-messages-include` directive, it would fail correct (I mean, without this PR)?

Is there a reason to do it as a document fragment test rather than one of the standard transclusion methods listed in the issues?